### PR TITLE
Pricing page

### PR DESCRIPTION
**Pull request by dtuite**
Preview: https://deploy-preview-555--roadie.netlify.app/pricing/

### CODE REVIEW

**Comment by dtuite** [comment]
https://app.shortcut.com/larder/story/6156/add-a-pricing-page

File: src/components/Byline/Byline.test.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { expect } from 'chai';
 import React from 'react';
 import Enzyme, { shallow } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
-import sinon from 'sinon';
+import { useFakeTimers } from 'sinon';
```
Comment (dtuite) [inline src/components/Byline/Byline.test.js:5]: This is unrelated to the PR but linting was failing without this change.

File: src/components/CallToAction/FormSubmissionModal.js [hunk 2]
```diff
@@ -62,8 +66,10 @@ const FormSubmissionModal = ({
     </span>
   ),
 }) => {
-  let followOnContent = <NewsletterAndTwitterInner siteMetadata={siteMetadata} />;
-  if (followOn === 'TWITTER') {
+  let followOnContent;
+  if (followOn === 'NEWSLETTER_AND_TWITTER') {
+    followOnContent = <NewsletterAndTwitterInner siteMetadata={siteMetadata} />;
+  } else if (followOn === 'TWITTER') {
     // Doesn't make sense to offer to let people sign up to the newsletter immediately after
```
Comment (dtuite) [inline src/components/CallToAction/FormSubmissionModal.js:73]: The changes in this file basically make it possible to not have any "follow on" text in the modal.

File: src/components/landing/CustomerLogoCloud.js [hunk 3]
```diff
@@ -58,14 +58,12 @@ const LOGOS = [{
...
-      <source srcSet={src.png} type="image/png" />
-      <img src={src.png} {...rest} className="h-8 sm:h-10" />
-    </picture>
-  </div>
+export const LogoItem = ({ src, ...rest }) => (
+  <picture>
+    <source srcSet={src.webp} type="image/webp" />
+    <source srcSet={src.png} type="image/png" />
+    <img src={src.png} {...rest} />
+  </picture>
```
Comment (dtuite) [inline src/components/landing/CustomerLogoCloud.js:66]: These changes allow me to re-use this component in the sidebar of the pricing pages.

File: src/components/pricing/TeamPricingTier.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            color="primary"
+            text="Request early access"
+            to="/pricing/teams/"
+          />
+        </div>
+      </div>
+
+      <TierLimitations
+        limitations={[
+          'Up to 5 software components tracked.',
```
Comment (sblausten) [inline src/components/pricing/TeamPricingTier.js:33]: Feels like too few for a team using a microservices approach? imo 10 would be better

**Review by sblausten — APPROVED** [review]
Looks great! One idea for an iteration is to make clear on the enterprise pricing column that there is a free trial also.

File: src/components/pricing/TeamPricingTier.js (hunk 4 above)
Comment (dtuite) [reply src/components/pricing/TeamPricingTier.js:33]: Yeah maybe. It's easy to change anyway.

**Pull request merged**